component-maps.yml

```diff
@@ -288,6 +288,12 @@ git:
     release_component: false
     independent_component: true
 
+  app-update-module:
+    docker_image: []
+    docker_container: []
+    release_component: true
+    independent_component: true
+
```

Comment:
I feel like Lluis is the expert here, I would defer to him, unless he does not have the time, then I will review (it will take me much, much longer)